mkdocs.yml

```diff
@@ -0,0 +1,123 @@
+# Project information
+site_name: 'React UI docs'
+site_description: 'React UI docs.'
```

Comment:
Improve the description as it is used by Google and other search bots.

Reply:
fixed, texts come from the old site.